database/definition-variablen.txt

```diff
@@ -0,0 +1,84 @@
+Variable
+
+# Approved
```

Comment:
I find these 4 states confusing. Can we name it "status" and define the states "online", "missinginfo", "cancelled", and "" which practically equals "missinginfo"?

Reply:
I like this idea, it is more understanding, I agree. What is the best flow to change this @weeman1337 ? I can update the database accordingly but would like to coordinate this with you so the new information can be understood from the system.